app/screens/channel/header/header.tsx

```diff
@@ -124,20 +125,26 @@ const ChannelHeader = ({
             return;
         }
 
+        // When calls is enabled, we need space to move the "Copy Link" from a button to an option
+        const height = QUICK_OPTIONS_HEIGHT + (callsEnabled ? 48 : 0);
```

Comment:
48 is a magic number. Not sure, but you can probably import `ITEM_HEIGHT` from `components/option_Item` (if it is the correct one to have).

Reply:
Ah, there it was. Thanks!